pkg/installation/semver/version.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package semver is a wrapper for handling of semantic version values.
+package semver
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	k8sver "k8s.io/apimachinery/pkg/util/version"
+)
+
+// Version describes a semantic version value
+type Version k8sver.Version
+
+// String returns string representation of a semantic version
+// (https://semver.org) value with leading 'v' character.
+func (v Version) String() string {
+	vv := k8sver.Version(v)
+	s := (&vv).String()
+	if !strings.HasPrefix(s, "v") {
+		s = "v" + s
+	}
+	return s
+}
+
+// Parse parses a semantic version (https://semver.org) value with leading 'v'
+// character.
+func Parse(s string) (Version, error) {
+	var vv Version
+	if !strings.HasPrefix(s, "v") {
```

Comment:
We could be lenient here and just pass through the semver, as k8sver understands an optional `v` prefix.

Reply:
I think we should be less lenient first, then maybe more lenient.
Right now all plugins have v* prefix, so let's keep it?

Reply:
IMHO the k8sver handles errors just fine, so why add additional checks around that call? I don't think we will switch to another SemVer implementation which would require the v prefix (_that_ would justify the additional check).
But it's a minor thing, I'm fine either way. Your call.